setup.py

```diff
@@ -1,19 +1,33 @@
-"""
-This is a setup.py script generated by py2applet
+from setuptools import setup, find_packages
 
-Usage:
-    python setup.py py2app
-"""
-
-from setuptools import setup
-
-APP = ['dashboard.py']
-DATA_FILES = []
-OPTIONS = {}
+NAME = 'OpenEIT_Dashboard'
+VERSION = '0.1.0'
+AUTHOR = 'Jean Rintoul'
+LICENSE = 'LICENSE'
+INSTALL_REQUIRES = [
```

Comment:
Same problem as in requirements.txt

Reply:
responded to `requirements.txt` problem above.